EagleThreadBot/SlashCommands/TagCommand.cs

```diff
@@ -1,40 +1,37 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
-using System.Net.Http;
 using System.Threading.Tasks;
 
 using DSharpPlus;
 using DSharpPlus.SlashCommands;
 
 using EagleThreadBot.Common;
 
-using Newtonsoft.Json;
-
 namespace EagleThreadBot.SlashCommands
 {
 	public class TagCommand : ApplicationCommandModule
```

Comment:
either mark this with ModuleLifespan.Transient or fetch the tag list from Program#GetTagList every call to avoid stale data

that shouldn't be too intensive since effectively you're just copying 8 bytes of data (one memory address pointer)

Reply:
to clarify: fetching every call shouldnt be too intensive, you can mark the command as ModuleLifespan.Singleton if you want